lib/views/background_sound/components/custom_track_shape_component.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:Medito/constants/constants.dart';
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class CustomTrackShapeComponent extends RectangularSliderTrackShape {
```

Comment:
I think we have multiple TrackShapes in the project now, so maybe calling it something like "BackgroundSoundVolumeTrackShape" would be better?

Reply:
Yes, you're right